environment.yml

```diff
@@ -6,7 +6,7 @@ dependencies:
     - python=3.12
     # Required dependencies
     - gmt=6.5.0
-    - ghotscript=10.03.0
+    - ghostscript=10.03.0
```

Comment:
Do we actually need to pin ghostscript in environment.yml? This line was added in #3112, but we didn't set any pin for ghostscript before.

Reply:
Better to pin ghostscript here. Otherwise, new developers may install a different gs version and the tests may fail due to minor changes in gs.